library/entropy_poll.c

```diff
@@ -50,28 +50,33 @@
 
 #include <windows.h>
 #if _WIN32_WINNT >= 0x0501 /* _WIN32_WINNT_WINXP */
```

Comment:
(sorry, just noticed) Since we no longer support <= WINXP, can we delete these checks?